scrapyd_k8s/launcher/docker.py

```diff
@@ -55,36 +60,44 @@ def schedule(self, project, version, spider, job_id, settings, args):
         )
 
     def cancel(self, project_id, job_id, signal):
-        c = self._get_container(project_id, job_id)
-        if not c:
+        container = self._get_container(project_id, job_id)
+        if not container:
             return None
 
-        prevstate = self._docker_to_scrapyd_status(c.status)
-        if c.status == 'created' or c.status == 'scheduled':
-            c.remove()
-        elif c.status == 'running':
-            c.kill(signal='SIG' + signal)
+        prevstate = self._docker_to_scrapyd_status(container.status)
+        if container.status == 'created' or container.status == 'scheduled':
+            container.remove()
+        elif container.status == 'running':
+            container.kill(signal='SIG' + signal)
```

Comment:
This is a change in code-style, I'd prefer to keep it as `c`.
Yes, often quite indicative names are often a good idea. But in this case, it is such a common domain object here that I would think it's obvious, and provide an easier to read experience (terser).
And now there are different code styles in the same project / file, which is more confusing.
So unless you have important objections, please revert.

Reply:
Let's keep it consistent in the project. If we are going the change the code style, we should indeed do it for the entire project. But that should not be part of this PR!